pineappl_cli/src/evolve.rs

```diff
@@ -70,6 +70,7 @@ mod eko {
         V0(MetadataV0),
         V1(MetadataV1),
         V2(MetadataV2),
+        V3(MetadataV3),
```

Comment:
@felixhekhorn would probably say that we should explicitly encode the EKO version here, and I tend to agree. However, I didn't do that in the past, because I didn't know what EKO version ranges `V0`, `V1` and `V2` correspond to, so let's at least note that here by adding a comment:

    // v0.15 - v????

When we know all the ranges we change that in separate PR.

Reply:
I also do not know the exact ranges so I simply added a comment as you suggested.